react/tic-tac-toe/src/app/screens/Login/fields/index.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable react/no-multi-comp */
+import React from 'react';
+
+import styles from '../styles.module.scss';
+
+export const userInput = props => {
```

Comment:
`userInput` and `passInput` should be separated into a file per component.
Also, remember components name should be capitalized (`UserInput`) and defined as regular functions, not arrow.
https://github.com/Wolox/tech-guides/blob/tech-refactor/frontend/react/style-guide.md#class-vs-reactcreateclass-vs-stateless